frontend/app/src/components/helper/display/icons/AssetIcon.vue

```diff
@@ -99,7 +100,8 @@ export default defineComponent({
         return `/assets/images/defi/weth.svg`;
       }
       const url = `${api.serverUrl}/api/1/assets/${id}/icon`;
-      return get(changeable) ? `${url}?t=${Date.now()}` : url;
+      const currentTimestamp = get(timestamp) || Date.now();
```

Comment:
What is the advantage of this vs just using `Date.now()`?

Reply:
Date.now() is used as a random string to append to the URL.
This is to get the latest image.
get(timestamp) here is used to send some kind of signal from parent component (when icon is refreshed) to tell this component to refetch the image.